content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```diff
@@ -26,11 +33,11 @@ spec:
       # Tempo service name is "query-frontend" and is in the "tempo" namespace.
       # Make sure the URL you provide corresponds to the non-GRPC enabled endpoint
       # It does not support grpc yet, so make sure "use_grpc" is set to false.
-      in_cluster_url: "http://query-frontend.tempo:3200"
+      in_cluster_url: "http://tempo-tempo-query-frontend.tempo.svc.cluster.local:3200/"
       provider: "tempo"
       use_grpc: false
-      # Public facing URL of Grafana 
-      url: "http://my-tempo-host:3200"
+      # Public facing URL of Tempo 
+      url: "https://tempo-tempo-query-frontend-tempo.apps-crc.testing/"
 ```
 
 The default UI for Grafana Tempo is Grafana, so we should also set the Grafana url in the configuration: 
```

Comment:
Mention this is just an example (so people don't think they need to copy this exactly) and capitalize "URL".
```suggestion
The default UI for Grafana Tempo is Grafana, so we should also set the Grafana URL in the Kiali configuration, such as this example: 
```